vhdbuilder/packer/init-variables.sh

```diff
@@ -82,10 +82,6 @@ fi
 if [ -z "${VNET_RG_NAME}" ]; then
 	if [ "$MODE" == "linuxVhdMode" ]; then
 		VNET_RG_NAME="nodesig-${ENVIRONMENT}-${PACKER_BUILD_LOCATION}-agent-pool"
-		if [ "${ENVIRONMENT,,}" == "prod" ]; then
```

Comment:
I missed this originally, which is why the previous build run against this branch failed. Rerunning with this change as well.